pytensor/compile/function/types.py

```diff
@@ -1106,6 +1106,18 @@ def sync_shared(self):
         # NOTE: sync was needed on old gpu backend
         pass
 
+    def dprint(self, **kwargs):
+        """Debug print itself
+
+        Parameters
+        ----------
+        kwargs:
+            Optional keyword arguments to pass to debugprint function.
+        """
```

Comment:
In general do you know of an elegant way to copy over docstrings? It'd be nice if we could see the full dprint docstring, but only if we don't have to re-copy it in every helper method.

Reply:
I don't know a way but I suspect there might be one.

Reply:
Internet shows stuff with assigning `.__doc__` variables. Feels a bit hacky to me but maybe it's conventional. Checking if @OriolAbril has any input here

Reply:
I have seen and used copying things over and setting the `.__doc__` attribute manually, I don't think there is anything wrong with it